advocacy_docs/supported-open-source/patroni/migrating.mdx

```diff
@@ -0,0 +1,323 @@
+
+For sake of simplicity we will put some notes, when relevant, considering 3 possibilities for an existing cluster, just so we do not need to repeat ourselves regarding similar steps:
+
+1. Managed by [EFM](/efm/latest/).
+2. Managed by [repmgr](/repmgr/latest).
+3. With no HA tool.
+
+For the example, the following assumptions are made:
+
+1. You already have an working `etcd` cluster &mdash; refer to [Quick start on RHEL 8 - 1. Etcd](rhel8_quick_start#1-etcd)/[Quick start on Debian 11 - 1. Etcd](debian11_quick_start#1-etcd) for instructions on how to set up an `etcd` cluster. The `etcd` cluster here is composed of the following nodes:
+    1. `etcd-node-1`.
+    2. `etcd-node-2`.
+    3. `etcd-node-3`.
+2. You have a Postgres cluster that needs to be migrated, which is composed of the following nodes:
+    1. `postgres-node-1` &mdash; the primary node.
+    2. `postgres-node-2` &mdash; a standby of `postgres-node-1`.
+    3. `postgres-node-3` &mdash; a standby of `postgres-node-1`.
+3. The existing Postgres cluster uses replication slots to control replication from primary to standbys, and they are named:
+    1. `slot_for_postgres_node_2` &mdash; slot used by node `postgres-node-2`.
+    2. `slot_for_postgres_node_3` &mdash; slot used by node `postgres-node-3`.
+4. You already have a few Postgres users in the cluster:
+    1. `postgres` with password `postgres_password` &mdash; Postgres superuser.
+    2. `replicator` with password `replicator_password` &mdash; used by replication connections.
+    3. `rewind` with password `rewind_password` &mdash; user with rights for executing `pg_rewind`.
+5. You are running Postgres 15 with the following characteristics:
+    1. Postgres is running on port `5432`.
+    2. Postgres data directory is `/var/lib/pgsql/15/data`.
+    3. Postgres binaries directory is `/usr/pgsql-15/bin`.
+6. You already have installed Patroni in the Postgres nodes.
+7. The systemd unit names are:
+    1. `postgres` &mdash; for Postgres.
+    2. `patroni` &mdash; for Patroni.
+    3. `edb-efm` &mdash; in case EFM is being used in the existing cluster.
+    4. `repmgrd` &mdash; in case repmgr is being used in the existing cluster.
+8. The configuration files are:
+    1. `/etc/patroni.yml` &mdash; for Patroni.
+    2. `/etc/repmgr.conf` &mdash; in case repmgr is being used in the existing cluster.
+9. The cluster name is `test_cluster`. This is relevant only for EFM and Patroni.
+
+**Notes:**
+
+* Please adapt the example names according to your actual cluster.
+* Setting up Watchdog and HAProxy are out of the scope of this example. However, as they are part of the recommended architecture, we recommend you read the following documentation:
+    * About Watchdog:
+        * [Quick start on RHEL 8 - 3. Watchdog](rhel8_quick_start#3-watchdog)/[Quick start on Debian 11 - 3. Watchdog](debian11_quick_start#3-watchdog) &mdash; set up an Watchdog device.
```

Comment:
It would be easier to simplify the link text to avoid changing this page too often if we change the QS.

Suggest:
`Quick start on [RHEL 8](...) or [Debian 11](...): set up a watchdog device`

Also, don't use capital letters for Watchdog when it's in the sentence (there are a few of them here)